This fixes the following build error:

    [869/876] Linking CXX shared library lib/liblldb.so.5.0.0
    FAILED: lib/liblldb.so.5.0.0
    ...
    /usr/bin/../lib/gcc/x86_64-alpine-linux-musl/6.4.0/../../../libclangSema.a(SemaExprObjC.cpp.o): In function `applyCocoaAPICheck(clang::Sema&, clang::ObjCMessageExpr const*, unsigned int, bool (*)(clang::ObjCMessageExpr const*, clang::NSAPI const&, clang::edit::Commit&)) [clone .constprop.696]':
    SemaExprObjC.cpp:(.text._ZL18applyCocoaAPICheckRN5clang4SemaEPKNS_15ObjCMessageExprEjPFbS4_RKNS_5NSAPIERNS_4edit6CommitEE.constprop.696+0x13c): undefined reference to `clang::edit::rewriteObjCRedundantCallWithLiteral(clang::ObjCMessageExpr const*, clang::NSAPI const&, clang::edit::Commit&)'
    /usr/bin/../lib/gcc/x86_64-alpine-linux-musl/6.4.0/../../../libclangSema.a(SemaExprObjC.cpp.o): In function `(anonymous namespace)::ARCCastChecker::VisitCallExpr(clang::CallExpr*)':
    SemaExprObjC.cpp:(.text._ZN12_GLOBAL__N_114ARCCastChecker13VisitCallExprEPN5clang8CallExprE+0x12e): undefined reference to `clang::ento::coreFoundation::followsCreateRule(clang::FunctionDecl const*)'
    clang-5.0: error: linker command failed with exit code 1 (use -v to see invocation)

See https://github.com/alpinelinux/aports/pull/2342 for more information.
--- a/cmake/modules/LLDBStandalone.cmake.orig
+++ b/cmake/modules/LLDBStandalone.cmake
@@ -101,6 +101,7 @@
 
   # Import CMake library targets from LLVM and Clang.
   include("${LLVM_OBJ_ROOT}/lib${LLVM_LIBDIR_SUFFIX}/cmake/llvm/LLVMConfig.cmake")
+  include("/usr/lib/cmake/clang/ClangConfig.cmake")
   if (EXISTS "${LLVM_OBJ_ROOT}/lib${LLVM_LIBDIR_SUFFIX}/cmake/clang/ClangConfig.cmake")
     include("${LLVM_OBJ_ROOT}/lib${LLVM_LIBDIR_SUFFIX}/cmake/clang/ClangConfig.cmake")
   endif()

